runtime-common/stdlib/msgpack/packer.h

```diff
@@ -68,14 +69,14 @@ class packer : private vk::not_copyable {
 class packer_float32_decorator {
 public:
   static void clear() noexcept {
-    serialize_as_float32_ = 0;
+    SerializationLibContext::get().serialize_as_float32_ = 0;
   }
 
   template<class StreamT, class T>
   static void pack_value_float32(packer<StreamT> &packer, const T &value) {
-    ++serialize_as_float32_;
+    ++SerializationLibContext::get().serialize_as_float32_;
```

Comment:
let's save an instance of `SerializationLibContext` into a variable rather than calling its `get` method twice